sbndqm/dqmAnalysis/CRT/SBND/fcl/config_crt_dqm.fcl

```diff
@@ -1,47 +1,177 @@
+	#display_range: []
+      }
+      T0clockdrift: {
+	units: ns
+	title: "FEB %(instance)s"
+	display_range: [-20.0, 20.0]
+      }
+      T1ResetTDCDiff: {
+	units: ns
+	title: "FEB %(instance)s"
+	display_range: [0.0, 100.0]
+      }
+      MaxADCValue: {
+	units: ADC
+	title: "FEB %(instance)s"
+	display_range: [0.0, 4000.0]
+      }
+      MaxADCChannel: {
+	title: "FEB %(instance)s"
+	display_range: [0.0, 31.0]
+      }
+      MaxADCValuePair: {
+	units: ADC
+	title: "FEB %(instance)s"
+	display_range: [0.0, 4000.0]
+      }
+      MaxADCChannelPair: {
+	title: "FEB %(instance)s"
+	display_range: [0.0, 31.0]
+      }
+      Flag: {
+	title: "FEB %(instance)s"
+	display_range: [0.0, 15.0]
+      }
+      baseline: {
+	units: ADC
+	title: "FEB %(instance)s"
+	display_range: [0.0, 4000.0]
+      } 
+      AverageADC: { 
+	units: ADC
+	title: "FEB %(instance)s"
+	display_range: [0.0, 4000.0] 
+      }    
+      Flag3Hit: {
+	title: "FEB %(instance)s"
+      }
+      Deadtime: {
+	units: ns
+	title: "FEB %(instance)s"
+	#display_range: [0.0, 1000000.0]
+      }
+      ReadoutRate: {
+	title: "FEB %(instance)s"
+      }
+      PullWindow: {
+	units: ns
+	title: "FEB %(instance)s"
+      }
+      earlysynch: { 
+	units: ns
+	title: "FEB %(instance)s"
+	display_range: [-100000,100000] 
+	alarmrange: [-100000,100000] 
+      }
+      latesynch: { 
+	units: ns
+	title: "FEB %(instance)s"
+	display_range: [-100000,100000]
+	alarmrange: [-100000,100000] 
+      }
+      MissingT0: {
+	title: "FEB %(instance)s"
+      }
+      MissingT1: {
+	title: "FEB %(instance)s"
+      }
+      NT0Resets: {
+	title: "FEB %(instance)s"
+      }
+      NT1Resets: {
+	title: "FEB %(instance)s"
+      }
     }
   }
-  metric_channel_config: {
-    groups: { CRT_channel:[[0,7616]] }
-    streams: [archiving]
-    metrics:{ ADC:{ display_range: [0, 4000] }
-              lastbighit:{}
-    	  pedestalMean:{}
-              pedestalRMS:{}
-    	  pedestalRMS2:{}
-              ChFlag3Rate:{}
+  
+  metric_channel_config: 
+  {
+    groups: { CRT_channel: @local::crt_channel_list }
+    streams: [slow, fast, archiving]
+    metrics:{ 
+      ADC:{ 
+	units: ADC
+	title: "CRT channel %(instance)s"
+	display_range: [0, 4000] 
+      }
+      lastbighit: {
+	units: ns
+	title: "CRT channel %(instance)s"
+	display_range: [0.0, 1000000000.0]
+      }
+      ChReadoutRate: {
+	title: "CRT channel %(instance)s"
+      }
+      Pedestal: {
+	units: ADC
+	title: "CRT channel %(instance)s"
+	display_range: [0.0, 4000.0]
+      }
     }       
   }
+
   metric_event_config: {
-    groups: { }
-    streams:[archiving]
-    metrics:{ }
+    groups: { CRT_event: [0] }
+    streams:[slow, fast, archiving]
+    metrics:{ 
+      T0ResetSpread: {
+	units: ns
+	#title: "Maximum difference in t0 reset timestamps"
+      }
+      T1ResetSpread: {
+	units: ns
+	#title: "Maximum difference in t1 reset timestamps"
+      }
+    }
   }
+  
+  # TODO - remove this
   metric_fragment_config: {
```

Comment:
There is a general mismatch between
a) The metrics listed in the fcl
b) The metrics listed in the comments at the top of the module
c) The metrics actually sent

For example there are fragment level metrics described in the module.

Reply:
Fair point, I'd neglected to do that. Onto cleanup! Thanks for the shout